.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "pip" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: "deps: "
+    labels:
+      - "run-integration-test"
+      - "evaluation requested"
```

Comment:
lets not add them by default. Otherwise it will start consuming alot of aicore requests and tokens. What do you think?

Reply:
removed them.